velox/vector/tests/utils/VectorTestBase.h

```diff
@@ -459,6 +459,21 @@ class VectorTestBase {
         size, sizeAt, valueAt, isNullAt, valueIsNullAt, arrayType);
   }
 
+  /// Similar to makeArrayVectorFromJson, Creates an ArrayVector from list of
+  /// JSON arrays of arrays.
+  /// @tparam T Type of array elements. Must be an integer: int8_t, int16_t,
```

Comment:
Why are these restricted only to integers ?

Reply:
added that in to keep it consistent to makeArrayVectorFromJson function which has the same restriction. If it makes better sense to leave it open to to all types, I could add some testcases with strings and remove this

Reply:
@soumiiow Thanks. CI is red though. Please, take a look.